siguza-utils.mk

```diff
@@ -0,0 +1,85 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUPROJECTS      += siguza-utils
```

Comment:
No need to have a LIBS var for just one tool. Just put the libs/frameworks after the ldflags.